scout/server/blueprints/institutes/views.py

```diff
@@ -144,10 +173,12 @@ def institute_settings(institute_id):
         return redirect(request.referrer)
 
     institute_obj = store.institute(institute_id)
-    form = InstituteForm(request.form)
+    institute_form = InstituteForm(request.form)
+
+    beacon_form = controllers.populate_beacon_form(institute_obj)
 
     # if institute is to be updated
-    if request.method == "POST" and form.validate_on_submit():
+    if request.method == "POST" and beacon_form.validate_on_submit():
```

Comment:
I'm not sure I follow this; why replace the form.validate - didn't you rather want to add the beacon_form.validate?

Reply:
Good catch, this is a typo. I meant to write institute_form.validate_on_submit():

Reply:
When you click on submit on the beacon form you don't POST a request to this page, but to another endpoint in the views (`overview.add_beacon_datase`t)